web_app/models/IC_model2.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+import streamlit as st
+import numpy as np
+from . immersed_corrosion_models import thermo_nutrient_variability_steel_corrosion_model
+    
+
```

Comment:
All of the models should reference the original paper.

Reply:
Has this been fixed?

Reply:
yes